### DISCUSSION TITLE
Some observations and experience as a fresh user

### DISCUSSION THREAD

**Discussion by Ibrahim-Halil-Kuray** [ProxeusApp/community#15] [General]
I recently started as intern full stack develeoper at Proxeus Association. This is my [linkedin,](https://www.linkedin.com/in/ibrahim-halil-kuray/) you can have a look for more information about me. My first task is to explore the platform and learn its features. Here are my initial observations as a fresh Proxeus User :-)

**Comment by Ibrahim-Halil-Kuray** [comment]
An error occurred when I tried to add an ETH wallet address. "Please make sure your Metamask wallet has access."  When I refreshed the page and tried again, the trial was successful.

**Reply by loleg** [reply]
The new release aims to fix [issue #285](https://github.com/ProxeusApp/proxeus-core/issues/285) which is related to this problem. Please try again with a fresh account on an up to date server.

**Comment by Ibrahim-Halil-Kuray** [comment]
Making drag and drop boxes clickable can improve user experience.

**Reply by loleg** [reply]
I think a screenshot would really help, I guess you are talking about the Workflow editor here? Currently when you click on one of the nodes, there is no panel or popup explaining what the node does. It would be great to add this suggestion as a [GitHub Issue](https://github.com/ProxeusApp/proxeus-core/issues), ideally along with a sketch of what you have in mind.

**Comment by Ibrahim-Halil-Kuray** [comment]
When I click on the Help / Support section, “You can find more detailed information on how to use the Proxeus
platform in the Handbook.” opens a page like.
It would be better if the handbook would be opened in a new tab instead.

**Reply by loleg** [reply]
Good idea! This could be a good "first issue" type fix for a beginner ;-)

**Comment by Ibrahim-Halil-Kuray** [comment]
The documentation is very distracting. It would be a good start to briefly explain the main lines of the project instead
of the current method of narration. Afterwards, it would be a more effective approach to direct the reader only to the
part that interests them, depending on the reason for being on the page.

**Reply by loleg** [reply]
Oh yeah. Proxeus used to have a website which explained the project well. You can still find it on [Internet Archive](https://web.archive.org/web/20201101022252/https://www.proxeus.com/). Much of this content should be incorporated into a revised Handbook. Thanks a lot for the feedback.

**Comment by Ibrahim-Halil-Kuray** [comment]
On Proxeus.org demo section is not a demo. It would be great if there would be an explanatory video and an
executable example for people to try.

**Reply by loleg** [reply]
Great. Would you mind recording a screencast and sharing it with us? ;-)

**Comment by Ibrahim-Halil-Kuray** [comment]
There are buttons on the page that I have trouble understanding what they do. Having an explanation appear on
hovering them can improve the user experience. (HTML5 title attr.)

**Reply by loleg** [reply]
Super idea. Here again a screenshot would be helpful, and I think this is a bit of HTML programming that would be a good "first issue" fix.

**Comment by Ibrahim-Halil-Kuray** [comment]
The current application is not compatible with the ETH network.

**Reply by loleg** [reply]
I believe we have addressed this in the latest release.

**Comment by Ibrahim-Halil-Kuray** [comment]
It is not possible to undo changes in forms. Properties can be deleted from form elements, but it is not possible to
add them again.
Instead, it is necessary to delete the element, then re-affect the form and thus reach the default properties, which is
quite negative for the user experience.

**Reply by loleg** [reply]
True. This is a good improvement request that belongs in a GitHub Issue.

**Comment by Ibrahim-Halil-Kuray** [comment]
Action part is not clear enough. When the user comes to this part of the page, the subject can be clarified by means
of a demonstration in addition to the explanation.

**Reply by loleg** [reply]
I could really use a screenshot or at least a URL to see what you mean here.

**Comment by Ibrahim-Halil-Kuray** [comment]
Some examples of forms, workflows and templates should be found on the site for free. Thus, it may be possible for the
user to understand and use the system more quickly.

**Reply by loleg** [reply]
That's a great idea! A CMS (like [Typo3](https://demo.typo3.org/)) usually has a demo site with some stock content. This would be very helpful to get the main concepts of Proxeus across in short time.
